.github/workflows/benchmark.yml

```diff
@@ -37,6 +37,10 @@ on:
 env:
   PRISMA_TELEMETRY_INFORMATION: 'prisma benchmark.yml'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow